### PR TITLE
[statviz] Reference authz test cases from test plan

### PR DESCRIPTION
**Pull request by pylipp**
[test plan](https://docs.google.com/spreadsheets/d/1sDhSsaVwNxAhGn1VYACDPVepZpvlwTkgy9nmeTaRy2Q/edit?pli=1#gid=1709672082)

### CODE REVIEW

**Comment by pylipp** [comment]
[2.0 StatViz BE testing](https://trello.com/c/mKNwn1wN/1391-20-statviz-be-testing)

File: back/test/endpoint_tests/test_statistics.py [hunk 1]
```diff
@@ -395,10 +399,12 @@ def test_authorization(read_only_client, mocker):
     ]:
         assert_forbidden_request(read_only_client, query)
 
+    # Test case 11.1.5
     # Base 5 does not exist
     query = "query { createdBoxes(baseId: 5) { facts { productId } } }"
     assert_forbidden_request(read_only_client, query)
```
Comment (aerinsol) [inline back/test/endpoint_tests/test_statistics.py:405]: @pylipp shouldn't this be an invalid data response rather than a forbidden response?

**Review by aerinsol — CHANGES_REQUESTED** [review]
Just a small clarification needed

File: back/test/endpoint_tests/test_statistics.py (hunk 1 above)
Comment (pylipp) [reply back/test/endpoint_tests/test_statistics.py:405]: @aerinsol my thoughts were: if a forbidden response is returned, it isnt even revealed that the base does not exists which I find more secure.
Also for checking invalid data a DB query is required but for authz check it's not.

Comment (aerinsol) [reply back/test/endpoint_tests/test_statistics.py:405]: I see, it's part of the series of tests for unauthorized access to show that now more information is shown than needed. Thanks for clarifying.

**Review by aerinsol — APPROVED** [review]

**Pull request merged**